From: Marc Kleine-Budde <mkl@pengutronix.de>
Subject: don't overwrite CFLAGS, use CPP flags during compilation

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
# rsc: 20091229: Dominik Brodowski said: "Applied, thanks!"
# wsa: 20091224: Sent upstream: http://lists.infradead.org/pipermail/linux-pcmcia/2009-December/006676.html
# wsa: 20091230: applied upstream \o/ : http://git.kernel.org/?p=linux/pcmcia/pcmciautils.git;a=commitdiff;h=d399c1fac7f640453c7dbd83ae459164686099de

 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: pcmciautils-015/Makefile
===================================================================
--- pcmciautils-015.orig/Makefile
+++ pcmciautils-015/Makefile
@@ -122,7 +122,7 @@ WARNINGS += $(call cc-supports,-Wno-poin
 WARNINGS += $(call cc-supports,-Wdeclaration-after-statement)
 WARNINGS += -Wshadow
 
-CFLAGS := -pipe -DPCMCIAUTILS_VERSION=\"$(VERSION)\"
+CFLAGS += -pipe -DPCMCIAUTILS_VERSION=\"$(VERSION)\"
 YFLAGS := -d
 
 HEADERS = \
@@ -213,7 +213,7 @@ ccdv:
 	@$(HOSTCC) -O1 build/ccdv.c -o build/ccdv
 
 .c.o:
-	$(QUIET) $(CC) $(CFLAGS) -c -o $@ $<
+	$(QUIET) $(CC) $(CFLAGS) $(CPPFLAGS) -c -o $@ $<
 
 %.c %.h : %.y
 	$(YACC) $(YFLAGS) $<
